oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/source/EBPFProfilingData.java

```diff
@@ -46,7 +47,7 @@ public String getEntityId() {
     private String taskId;
     private long uploadTime;
     private String stackIdList;
+    private EBPFProfilingTargetType targetType;
     private byte[] stacksBinary;
```

Comment:
I think you should change this field name too?

Reply:
Do you mean `stacksBinary`? updated.